cost-analyzer/templates/kubecost-admission-controller-service-template.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.kubecostAdmissionController -}}
+{{- if .Values.kubecostAdmissionController.enabled -}}
+apiVersion: v1
+kind: Service
+metadata:
+  name: webhook-server
```

Comment:
I assume we'll be revisiting this name in a future PR?

Reply:
this is baked into the SSL key. I'd have to regenerate everything...so we can revisit in a future PR.